app/modules/intelligence/tools/tool_schema.py

```
@@ -2,3 +1,4 @@
+from typing import Any, Dict, Type, Union
```

Ruff (0.8.2)

`typing.Type` imported but unused

Remove unused import: `typing.Type`

(F401)